test/extractors/chroma.js

```diff
@@ -0,0 +1,48 @@
+var chai = require('chai');
+var assert = chai.assert;
+var TestData = require('../TestData');
```

Comment:
I know we haven't done this for other tests yet, but since this is new, could we maybe use [meyda/gauge](https://github.com/meyda/gauge) to get the test data? it'd be a case of including `gauge` as a git submodule and `require`ing the JSON file

Reply:
Happy to look into this, but it may take some time to get around to.

I'm not especially a fan of git submodules, are you happy to pull it in as an `npm` dependency as an alternative? (Either by referencing the repo or publishing `gauge` to npm.) I can do either though!

Reply:
Thinking some more, I think it might it make more sense to move all the tests over at once in a separate PR. I'm hesitant about how conventions should be established in gauge as it seems so new.

Reply:
Yeah, it's a new thing and I only made it because it was useful for the (as of now incomplete) Rust port of meyda – which doesn't use npm of course.

I guess we can keep this here for now and do a proper refactor of tests later on.